### PR TITLE
Enfore Uppercase

### PR DESCRIPTION
**Pull request by TatsuyaIshibe**
- Listener is a pain in the ass to think about whether it's uppercase or lower case.

### CODE REVIEW

**Comment by TatsuyaIshibe** [comment]
@hourglasshoro @geeknees @SamuraiT  What do you think?

File: pages/quests/index.tsx [hunk 1]
```diff
@@ -54,6 +54,7 @@ const Quests: NextPage = () => {
                   <Input
                     placeholder="Keyword"
                     onChange={inputChange}
+                    textTransform="uppercase"
```
Comment (SamuraiT) [inline pages/quests/index.tsx:57]: uh good how about also deleting zenkaku space or hankaku space

**Review by SamuraiT — APPROVED** [review]

File: pages/quests/index.tsx (hunk 1 above)
Comment (hourglasshoro) [reply pages/quests/index.tsx:57]: Are Japanese keywords assumed?

Comment (TatsuyaIshibe) [reply pages/quests/index.tsx:57]: Japanese is not assumed.

**Review by geeknees — APPROVED** [review]
Considering the scene where the text is read out in a podcast, it makes sense to capitalize it.

**Pull request merged**